py_trees/blackboard.py

```diff
@@ -132,6 +132,19 @@ def get(self, name):
         except AttributeError:
             return None
 
+    def unset(self, name):
```

Comment:
I like this change. 

I believe you should call this method from ClearBlackboardVariable::initialise() as part of this PR instead of the delattr() call there.

Reply:
Makes sense. Done.

PTAL